### ISSUE TITLE
save_graphic does not keep the line breaks of texts on figure

### ISSUE THREAD

**Issue by dkobayas-cyber** [pyvista/pyvista#4355]
### Describe the bug, what's wrong, and what you expected.

I need to plot three lines of texts on a figure, and save the result with `save_graphic` function. However, the line breaks are not preserved when I save the figure. 
* The plot produced by the code, which is exactly right: 
<img src="https://user-images.githubusercontent.com/85417498/235216478-78ae63f7-58bf-40e0-9601-5befb9fa819c.png" width="400">

* A saved figure:  
<img src="https://user-images.githubusercontent.com/85417498/235216950-a70f74ea-2d6b-48d5-a12b-a4ad06b2a32d.svg" width="400">

### Steps to reproduce the bug.

```python
%matplotlib inline
from pyvista import set_plot_theme
set_plot_theme("document")

import pyvista as pv
from pyvista import examples

mesh = examples.download_bunny_coarse()
val1, val2, val3 = 1, 2, 3
font_size = 14
font = 'times'

pl = pv.Plotter(window_size=(700,700))
pl.add_mesh(mesh, show_edges=True, color="white")
pl.add_points(mesh.points, color="red", point_size=10)
pl.camera_position = [(0.02, 0.30, 0.73), (0.02, 0.03, -0.022), (-0.03, 0.94, -0.34)]
pl.add_text('(Text1: %d)\r\n(Text2: %d)\r\n(Text3: %d)'%(val1, val2, val3), 
               font_size=font_size, font=font)

pl.show(jupyter_backend='static')
pl.save_graphic('bunny.svg')
```

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Fri Apr 28 13:45:13 2023 EDT

                OS : Linux
            CPU(s) : 32
           Machine : x86_64
      Architecture : 64bit
               RAM : 503.6 GiB
       Environment : Jupyter
        GPU Vendor : NVIDIA Corporation
      GPU Renderer : NVIDIA A10/PCIe/SSE2
       GPU Version : 4.6.0 NVIDIA 515.48.07

  Python 3.8.5 (default, Sep  4 2020, 07:30:14)  [GCC 7.3.0]

           pyvista : 0.38.5
               vtk : 9.2.2
             numpy : 1.24.3
           imageio : 2.28.0
            scooby : 0.7.1
             pooch : v1.7.0
        matplotlib : 3.7.1
           IPython : 8.12.0
        ipywidgets : 7.6.5
         pythreejs : 2.4.2
             trame : 2.3.2
      trame_client : 2.7.5
      trame_server : 2.11.0
         trame_vtk : 2.4.4
      nest_asyncio : 1.5.6
--------------------------------------------------------------------------------
```

**Comment by akaszynski** [comment]
For the time being, you might have to create three text actors with `pl.add_text`

**Comment by dkobayas-cyber** [comment]
@akaszynski Thanks for your comment but since I'm a newbie to PyVista, I'm not sure how to use "text actors" for this code. Could you add a simple code snippet to show how to use that?

**Comment by dkobayas-cyber** [comment]
@akaszynski Strangely, the two texts get overlapped with each other. I changed several different `position` values, but the text does not change the position.

**Comment by dkobayas-cyber** [comment]
@akaszynski 
The problem has been solved! I was assuming `position` denotes the normalized position, but in fact it represents the pixel coordinates. My figure size is (700, 700), so I needed to set `position = (0.1, 660)` to show the text on the upper left corner. I need to adjust the position manually depending on the font size, so it's a bit inconvenient, though. The saved figure also looks alright after modification.
